modules/virt-customizing-storage-profile.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-customizing-storage-profile_{context}"]
+
+= Customizing the storage profile
+
+Customize your storage profile to define default settings for data volumes that you frequently allocate. You can create as many storage profiles as you want, associating each with a storage class. 
+
+Customizing a storage profile can be helpful if you have a storage class that is not recognized by the CDI, or recognized as unsupported. In this case, the administrator sets appropriate values in the storage profile to ensure successful allocations.
```

Comment:
Don't mention unsupported.  We don't determine that in CDI.  We just recognize certain provisioners or not.

Reply:
Got it.